### PR TITLE
Add backend supports for tso and scheduling

### PR DESCRIPTION
**Pull request by rleungx**
The component name might be changed later. Ref #1651

### CODE REVIEW

**Comment by rleungx** [comment]
@mornyx PTAL

**Comment by baurine** [comment]
I will help review next week. 

@rleungx , can you elaborate the PR when you're free, for example, how to build the dev env so I can test the PR, attach some screenshots. thanks~

**Comment by baurine** [comment]
test:

```
tiup playground nightly --pd.mode ms --pd.api 1 --pd.scheduling 1 --pd.tso 1 --pd.rm 0 --kv 3 --tiflash 0 --db 1 --host [10.2.8.101](http://10.2.8.101/)
```

**Review by baurine — COMMENTED** [review]
the code LGTM, let me test the compatibility with old version cluster

File: pkg/utils/topology/tso.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	"sort"
+
+	"github.com/pingcap/tidb-dashboard/pkg/pd"
+	"github.com/pingcap/tidb-dashboard/util/distro"
+	"github.com/pingcap/tidb-dashboard/util/netutil"
+)
+
+func FetchTSOTopology(ctx context.Context, pdClient *pd.Client) ([]TSOInfo, error) {
+	nodes := make([]TSOInfo, 0)
+	data, err := pdClient.WithoutPrefix().SendGetRequest("pd/api/v2/ms/members/tso")
```
Comment (baurine) [inline pkg/utils/topology/tso.go:17]: miss the `/` at the begining, else it reports the following error:

image

File: pkg/utils/topology/scheduling.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	"sort"
+
+	"github.com/pingcap/tidb-dashboard/pkg/pd"
+	"github.com/pingcap/tidb-dashboard/util/distro"
+	"github.com/pingcap/tidb-dashboard/util/netutil"
+)
+
+func FetchSchedulingTopology(ctx context.Context, pdClient *pd.Client) ([]SchedulingInfo, error) {
+	nodes := make([]SchedulingInfo, 0)
+	data, err := pdClient.WithoutPrefix().SendGetRequest("pd/api/v2/ms/members/scheduling")
```
Comment (baurine) [inline pkg/utils/topology/scheduling.go:17]: miss the `/` at the beginning, else it reports the following errors:

image

**Comment by baurine** [comment]
screenshot: 

image

image

image

image

**Comment by baurine** [comment]
test compatibility, connect with tidb 7.5.0 cluster, reports following error in the cluster info page:

img_v3_028v_c0ee8a24-2ba7-435b-8271-f41b0e31a16g

**Comment by baurine** [comment]
fix the following errors in commit: https://github.com/pingcap/tidb-dashboard/commit/c0d16250c0cded8bb7677a73f412f7ea3fcbb394

**Review by baurine — APPROVED** [review]

**Pull request merged**